pymobiledevice3/__main__.py

```diff
@@ -88,6 +88,9 @@ def cli():
         return
     except BrokenPipeError:
         traceback.print_exc()
+    except LaunchApplicationTimeoutError:
```

Comment:
The main try except handles more low level exception, you should it put it under webinspector

Reply:
changed to webinspector decorator